.travis.yml

```diff
@@ -1,9 +1,19 @@
 language: python
-python:
-  - "2.7"
-install: "python setup.py install"
-script: "py.test tests/unit"
 branches:
     only:
         - master
         - dogfood
+services:
+  - postgres
+python:
+  - "2.7"
+env:
+  - DB=sqlite
+  - DB=postgres
+install:
+  - sh -c "if [ $DB = postgres ]; then pip install psycopg2; fi"
+  - python setup.py install
+before_script:
+  - sh -c "if [ $DB = postgres ]; then psql -c 'CREATE DATABASE haas_travis;' -U postgres; fi"
+  - sh -c "if [ $DB = postgres ]; then cp ci/testsuite.cfg.postgres testsuite.cfg; fi"
+script: "py.test tests/unit tests/stress.py"
```

Comment:
To actually run these in parallel without interfering with eachother, they'd have to have their own postgres DB; I don't know that there's a way to do that without at least a little bit of hacking. With sqlite that isn't an issue since we're using an in-memory db (and it *should* work, though I haven't tested it), but sqlite is also a lot faster anyway.